fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -165,6 +159,12 @@ protected boolean shouldCacheProperties() {
         return extraPropertiesStore() != null && (!isReadonly() || this.propertiesDirectory != null);
     }
 
+
+    @Override
+    public boolean isRoot(final String id) {
```

Comment:
Because these functions are protected in the modeshape impl. In order to be used by the `ExternalJsonSidecarExtraPropertyStore` class, they must be public. While the `ExternalJsonSidecarExtraPropertyStore` class was part of the org.modeshape package, it could access those functions.